src/ml_gym/persistency/io.py

```diff
@@ -277,6 +283,38 @@ def get_experiment_statuses(self, grid_search_id: str) -> List[ExperimentStatus]
         response = GridSearchRestfulAPIClient._get_json_resource(url)
         experiment_statuses = [ExperimentStatus(**r) for r in response]
         return experiment_statuses
+    
+    def create_system_info(self) -> Dict:
```

Comment:
I think we should move this to a class like `Environment`, with a static function e.g. in the util package
https://github.com/mlgym/mlgym/tree/master/src/ml_gym/util

```python 
class Environment: 
    @static_method
    create_system_info() -> Dict:
        ....

```

Reply:
I think we should include all the package names (even if it's a long list, we can just filter it in the frontend) and the original python CMD call. Maybe we could check the entire process command?